tests/test_functors.py

```diff
@@ -174,12 +236,12 @@ def testComposite(self):
                                                'modelfit_CModel')}
 
         func2 = CompositeFunctor(funcDict2, filt=filt)
-        df2 = self._compositeFuncVal(func2)
+        df2 = self._compositeFuncVal(func2, parq)
         self.assertTrue(df.equals(df2))
 
         func2.filt = 'HSC-R'
-        df3 = self._compositeFuncVal(func2)
-        self.assertFalse(df2.equals(df3))
+        df3 = self._compositeFuncVal(func2, parq)
+        self.assertTrue(df2.equals(df3))
```

Comment:
This also changed from `assertFalse` to `assertTrue`...why? I don't remember why it was so, but should make sure that this isn't evaluating to true because of identical simulated values.  This probably means that generally it would be better to make sure that simulated values in each column are different, this way equality/inequality tests will be more useful.

Reply:
There's a switch of the filter value just above and since the values are the same the test is now True. I've done a similar step of modifying the value for specifically R band this time force the test to fail.